indra/tests/test_statements_serialization.py

```diff
@@ -1,88 +1,99 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
```

Comment:
I am, but we continue to drag it along behind us like an anchor :) I look forward to the day when we can jettison all of the extra code I wrote to keep everything 2/3 compatible.